src/assets/scripts/client/ui/TutorialView.js

```diff
@@ -299,10 +298,9 @@ export default class TutorialView {
         });
 
         this.tutorial_step({
-            title: 'Takeoff, part 1',
-            text: ['When it appears at the start of runway ({RUNWAY}) (which may take a couple of seconds), click it (or press the up arrow once)',
-                   'and type in "caf" (for "cleared as filed"). This tells the aircraft it is cleared to follow its flightplan.',
-                   'Just as in real life, this step must be done before clearing the aircraft for takeoff, so they know where they\'re supposed to go.'
+            title: 'Taxiing',
+            text: ['Now type in "taxi {RUNWAY}" into the command box after the callsign and hit',
+                   'Return; the aircraft should appear on the radar scope after a 3 or so seconds'
```

Comment:
Phrasing: "after a 3 or so seconds"

Reply:
Sorry I see the typo I'll have that done :)